src/prebid.js

```diff
@@ -32,8 +32,10 @@ var eventValidators = {
 
 /* Public vars */
 
+pbjs._auctionRunning = false;
```

Comment:
do we want to put this in `pbjs`?  They can reset this flag to force a new request without clearing the other globals...